appveyor.yml

```diff
@@ -23,9 +23,9 @@ cache:
   - C:\projects\integrations-core\embedded
 services:
   - iis
-  # - mssql2008r2sp2 # FIXME: re-enable when we can launch multiple mssql instances at once
-  - mssql2012sp1
-  # - mssql2014 # FIXME: re-enable when we can launch multiple mssql instances at once
+  # We can't start more than one MSSQL instance to avoid conflicts on TCP port 1433. The only workaround to test a check
```

Comment:
This comment can be understood here because we see the git diff. You are basically answering the FIXME that you are removing. However, it is not easy to understand when you don't see what was there before. Could you rephrase this comment with more context?